.github/workflows/lint_markdown.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Vale
         uses: errata-ai/vale-action@v2
         with:
-          vale_flags: "--glob='!{plugins/.snapshots/*,CHANGELOG.md,.github/styles/proselint/README.md}'"
+          vale_flags: "--glob=!{plugins/.snapshots/*,CHANGELOG.md,.github/styles/proselint/README.md}"
```

Comment:
Shouldn't it be
```suggestion
          vale_flags: "--glob={plugins/.snapshots/*,CHANGELOG.md,.github/styles/proselint/README.md}"
```

Reply:
Good question, `--glob` says which files to lint, so if we want to ignore files we need the `!`